### DISCUSSION TITLE
posts/2022-my/java/%E6%B3%9B%E5%9E%8B%E5%BC%BA%E5%8C%96

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#280] [Announcements]
# posts/2022-my/java/%E6%B3%9B%E5%9E%8B%E5%BC%BA%E5%8C%96

人若无名，专心练剑

http://www.iocaop.com/posts/2022-my/java/%E6%B3%9B%E5%9E%8B%E5%BC%BA%E5%8C%96.html

**Comment by lzcxfz** [comment]
可以，这个屎黄色背景我很喜欢
